Source/ListItemFormatter.swift

```diff
@@ -24,60 +24,116 @@ import Foundation
 
 /// ListItemFormatter can be used to format variable-length lists of things in a
 ///  locale-sensitive manner, such as "Monday, Tuesday, Friday, and Saturday" (in
-///  English) versus "lundi, mardi, vendredi et samedi" (in French)
+///  English) versus "lundi, mardi, vendredi et samedi" (in French).
 @objc(LNListItemFormatter) public class ListItemFormatter: Formatter, NSSecureCoding {
 
-    /// TBC
+    /// The following constants specify predefined format modes for list items.
     @objc(LNListItemFormatterMode) public enum Mode: Int {
-        case standard, or, unit
+
+        /// Mode for a typical 'and' list of arbitrary placeholders.
+        case standard
+
+        /// Mode for a typical 'or' list of arbitrary placeholders.
+        case or
+
+        /// A format mode suitable for arbitrary placeholder units with no mode (i.e
+        ///  measurement units).
+        case unit
     }
 
-    /// TBC
+    /// The following constants specify predefined format styles for list items.
     @objc(LNListItemFormatterStyle) public enum Style: Int {
```

Comment:
[nits] Maybe it's worth mentioning `Mode` and `Style` are for constructing the keys like `standard`, `standardNarrow`, and `standardShort` used in the [plists](https://github.com/liamnichols/ListItemFormatter/blob/master/Resources/cldr_plists/en.plist)?

Reply:
My original intention was to not mention any specifics on the underlying implementation but thinking about it, I don't see no harm so I'll give it a go 👍